docs/tutorials/introduction.ipynb

```diff
@@ -304,16 +320,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "a[4]"
+    "ds['row', 1:2]"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "a[4:5]"
+    "Attributes are stored in a dictionary similar to coordinates.\n",
+    "The difference is that attributes are not required to match in operations.\n",
+    "Therefore we can perform the following operation without resulting in a error about a coordinate mismatch:"
```

Comment:
Have coordinates (matching) been explained yet? If you rely on a previous tutorial, why is there a need for explaining indexing here?